### PR TITLE
PLAT-141602: Fixed IncrementSlider issues

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fixed IncrementSlider styling to look good on all skins.

### Links
[//]: # (Related issues, references)
PLAT-141602

### Comments

### CODE REVIEW

File: IncrementSlider/IncrementSlider.module.less [hunk 1]
```diff
@@ -9,6 +10,28 @@
...
+		.incrementButton,
+		.slider {
+			transform: @agate-incrementslider-bg-transform;
+		}
+
+		.slider {
+			&::before {
+				content: none;
+			}
+		}
```
Comment (adrian-cocoara-lgp) [inline IncrementSlider/IncrementSlider.module.less:34]: Why is this needed? I commented this block locally and I see no difference

**Comment by adrian-cocoara-lgp** [comment]
Note to next reviewer: You might consider that vertical `IncrementSlider` is a little bit short on Carbon, Cobalt, Copper, Elecrto, Titanium skins. We solved this issue on #589, so once it's merged, everything should look fine

File: IncrementSlider/IncrementSlider.module.less (hunk 1 above)
Comment (stanca-pop-lgp) [reply IncrementSlider/IncrementSlider.module.less:34]: image
It might interfere with the other ::before. (see Cobalt night)

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

File: IncrementSlider/IncrementSlider.module.less (hunk 1 above)
Comment (adrian-cocoara-lgp) [reply IncrementSlider/IncrementSlider.module.less:34]: 👍

**Review by jeonghee27 — APPROVED** [review]
LGTM
